packages/storybook/stories/va-memorable-date-uswds.stories.jsx

```diff
@@ -78,6 +79,20 @@ const CustomValidationTemplate = ({ label, name, required, error, uswds, value }
         error prop to be dynamically set if the parameters are not met.
       </div>
       <div className="vads-u-margin-top--2">
+      <pre className="vads-u-font-size--sm vads-u-background-color--gray-lightest vads-u-padding--2"><code>
+const [dateVal, setDateVal] = useState(value);<br/>
+const [errorVal, setErrorVal] = useState(error);<br/>
+const today = new Date();<br/>
+// new Date as YYYY-MM-DD is giving the day prior to the day select<br/>
+// new Date as YYYY MM DD is giving the correct day selected<br/>
+const dateInput = new Date(dateVal.split('-').join(' '));<br/>
+function handleDateBlur() &#x7b;<br/>
+  if (dateInput &lt;= today) &#x7b;<br/>
+    setErrorVal('Date must be in the future');<br/>
+  &#x7d; else &#x7b;<br/>
+    setErrorVal('');<br/>
+  &#x7d;<br/>
+&#x7d;</code></pre>
```

Comment:
Inline code example is easier to view - also, test the fix in these "Custom Validation" stories
- https://1593-mem-date-fixes--60f9b557105290003b387cd5.chromatic.com/?path=/docs/components-va-memorable-date--custom-validation
- https://1593-mem-date-fixes--60f9b557105290003b387cd5.chromatic.com/?path=/docs/uswds-va-memorable-date--custom-validation